cpp/harvest/nacjd.cc

```diff
@@ -158,6 +171,32 @@ void ExtractExistingIDsFromMarc(MARC::Reader * const marc_reader, std::set<std::
     }
 }
 
+void ExtractOneIDFromWebsite(const unsigned nacjd_id, const std::string nacjd_path) {
+    std::string nacjd_title_path(nacjd_path + NACJD_TITLES_FILE_NAME);
```

Comment:
I don't really understand why we need 2 files here. It should be enough to have a nacjd_json_path (which could be renamed to output_path or json_path, since the whole program is related to nacjd).